pom.xml

```diff
@@ -211,13 +162,116 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>migrate-db</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
```

Comment:
I don't think all these build/plugins should be in the `migrate-db` profile.  For example, look how exam does it.  I'd be surprised if this started and built properly if you didn't run the build under the migrate-db profile.

https://github.com/SmarterApp/TDS_ExamService/blob/develop/service/pom.xml#L200